saleor/plugins/openid_connect/utils.py

```diff
@@ -412,7 +413,8 @@ def get_or_create_user_from_payload(
             defaults=defaults_create,
         )
 
-    if not user.is_active:  # it is true only if we fetch disabled user.
+    site_settings = Site.objects.get_current().settings
+    if not user.can_login(site_settings):  # it is true only if we fetch disabled user.
```

Comment:
@wcislo-saleor We'll need to adapt Cloud authorization to this change.

Reply:
@cmiacz @wcislo-saleor , please do not proceed yet with the adaptation, as there might be a small change to that logic of who can log in, I will contact you once this is conversation is solved.

Reply:
Please let us know when final solution is ready.